media-libs/t1lib/t1lib-5.1.2-r3.ebuild

```diff
@@ -1,11 +1,11 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI="8"
 
 inherit flag-o-matic libtool toolchain-funcs
 
-DESCRIPTION="A Type 1 Font Rasterizer Library for UNIX/X11"
+DESCRIPTION="Type 1 Font Rasterizer Library for UNIX/X11"
 HOMEPAGE="ftp://metalab.unc.edu/pub/Linux/libs/graphics/"
```

Comment:
I cannot open ftp sites with firefox, maybe we should instead use `https://www.ibiblio.org/pub/Linux/libs/graphics/t1lib-5.1.2.lsm` directly

Reply:
Sounds good to me!